libobs/obs-video.c

```diff
@@ -56,24 +56,31 @@ static uint64_t tick_sources(uint64_t cur_time, uint64_t last_time)
 	pthread_mutex_unlock(&obs->data.draw_callbacks_mutex);
 
 	/* ------------------------------------- */
-	/* call the tick function of each source */
+	/* get an array of all sources to tick   */
+
+	obs->data.sources_to_tick.num = 0;
```

Comment:
sorry for the additional nitpick, but you can just do `data->` in all these cases because there's already a pointer to the same at the top of the function. :sweat_smile:

Reply:
Oh wait it seems that someone (myself?) already did the same myself right above. Well that's embarrassing

Reply:
fixed it for all cases in `tick_sources` now